fix: Adds clarification about azure params (#151)
meshcloud/meshcloud-docs: docs/meshstack.azure.landing-zones.md

Blueprints must reside inside a Management Group. It is assumed it is in the same group as the group where to put the Subscriptions by default. If the Blueprint is located in another group it can be configuered here.

> Operators must ensure to create these management groups in the meshcloud AAD Tenant before configuring them for use in a Landing Zone.

Reviewer: ```suggestion
Operators must ensure to create these management groups in the meshcloud AAD Tenant before configuring them for use in a meshLanding Zone.
```

| projectIdentifier  | The project identifier                                            |
| subscriptionId     | The ID of the Azure Subscription associated with this meshProject |

> **Please Note:** that there are a few shortcomings when dealing with Azure parameters. Resource Group names and locations can not be directly parametrized as their format of reporting missing parameters is not compatible with the replicator.

Reviewer: ```suggestion
**Please Note:** there are some specialities to keep in mind when dealing with Azure parameters. Resource group names and locations can not be parameterised via meshStack.
```

> When parameter are marked as a static parameter in the Azure Panel, then they are not reported as missing parameters during project replication by the Azure API and thus can not be replaced anymore. This means a replaceable parameter should never be marked as static in the Azure Blueprint panel.

Reviewer: ```suggestion
When parameters are marked as static in the Azure Panel, they can not be replaced or overwritten. This means default parameter should never be marked as static in the Azure Blueprint panel.
```
